configure

```diff
@@ -661,7 +661,6 @@ infodir
 docdir
 oldincludedir
 includedir
-runstatedir
```

Comment:
I remember that we have talked about this and you said we removed this 'runstatedir' on purpose?

Reply:
Apart from this question/clarification, it looks good 🚢

Reply:
Yes, it was emerged due to running autoconf on Ubuntu and should be removed before merging, so we are removing it with this PR.